recipes/openvdb/8.1.0/conanfile.py

```diff
@@ -0,0 +1,202 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class OpenVDBConan(ConanFile):
+    name = "openvdb"
+    description = (
+        "OpenVDB is an open source C++ library comprising a novel hierarchical data"
+        "structure and a large suite of tools for the efficient storage and "
+        "manipulation of sparse volumetric data discretized on three-dimensional grids."
+    )
+    license = "MPL-2.0"
+    topics = ("voxel", "voxelizer", "volume-rendering", "fx")
+    homepage = "https://github.com/AcademySoftwareFoundation/openvdb"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_blosc": [True, False],
+        "with_zlib": [True, False],
+        "with_log4cplus": [True, False],
+        "simd": [None, "SSE42", "AVX"],
```

Comment:
```suggestion
        "simd": [None, "sse42", "avx"],
```

Use lower case, because we already have other project which offer AVX and SSE as option, but all them are lower case. Let's keep the same pattern.